rest/model/admin.go

```diff
@@ -950,15 +950,17 @@ func (a *APIRepoTrackerConfig) ToService() (interface{}, error) {
 }
 
 type APISchedulerConfig struct {
-	TaskFinder    APIString `json:"task_finder"`
-	HostAllocator APIString `json:"host_allocator"`
+	TaskFinder       APIString `json:"task_finder"`
+	HostAllocator    APIString `json:"host_allocator"`
+	FreeHostFraction float64   `json:"free_host_fraction"`
```

Comment:
why is this exposed to the user? can we expose it as an int <100 rather than as a float?